src/components/map.jsx

```diff
@@ -34,7 +35,13 @@ function Map({ events }) {
             position={[event.githubUsername.location.lat, event.githubUsername.location.long]}
           >
             <Popup>
-              {event.githubUsername._id}: {event.event}
+              <div className="flex flex-col gap-[5px]">
+                <h1 className="font-[600]">{event.githubUsername._id}</h1>
+                <span>Location - {event.githubUsername.location.provided}</span>
+                <span>Repo - {event.repoName}</span>
```

Comment:
For the map, you only get one point added no matter how many total events.  Does it update to the most recent event each time? Or is it showing the first event the user added at the start or the socket stream?  Should this be clear in the popup? wdyt?

Reply:
It updates to the most recent event. Having more than one point from the same location on the map will crowd it up

Reply:
Oh yes we definitely don't want additional points, just was askign about the popup text. That's perfect. Thank you!

Reply:
Oh. Alright. Thank you